### PR TITLE
Allow to change mapproxy tiles url

### PR DESCRIPTION
**Pull request by procrastinatio**
This [Demo](https://mf-geoadmin3.dev.bgdi.ch/mom_mapproxy_sources/?topic=schneesport&lang=fr&bgLayer=voidLayer&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bafu.wrz-wildruhezonen_portal,ch.swisstopo.swisstlm3d-wanderwege,ch.swisstopo.pixelkarte-farbe-pk50.noscale,ch.swisstopo.hangneigung-ueber_30,ch.swisstopo-karto.hangneigung,ch.bafu.wrz-jagdbanngebiete_select,ch.bazl.heliports-gebirgslandeplaetze,ch.swisstopo-karto.schneeschuhrouten,ch.swisstopo-karto.skitouren,ch.swisstopo.skitourenkarte-50.metadata,ch.bav.haltestellen-oev&layers_visibility=false,false,false,false,true,false,true,true,true,true,true,false,true&layers_timestamp=18641231,,,,,,,,,,,,&layers_opacity=1,1,1,1,0.85,0.5,0.2,0.6,1,0.8,0.8,1,1&catalogNodes=2352&lon=7.70568&lat=46.32534&elevation=6733&heading=359.700&pitch=-39.862) is using `mapproxy`tiles from https://wmts{s}.dev.bgdi.ch ([example](https://wmts20.dev.bgdi.ch)), where both layers `ch.swisstopo-karto.schneeschuhrouten`and `` ch.swisstopo-karto.skitouren`are configured.

Choose the right mapproxy source with:

`$ export MAPPROXY_URL=//wmts{s}.dev.bgdi.ch`

### CODE REVIEW

**Comment by oterral** [comment]
lgtm.  @loicgasser an opinion?

**Comment by oterral** [comment]
Work also on [int](https://mf-geoadmin3.int.bgdi.ch/mom_mapproxy_sources/?topic=schneesport&lang=fr&bgLayer=voidLayer&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bafu.wrz-wildruhezonen_portal,ch.swisstopo.swisstlm3d-wanderwege,ch.swisstopo.pixelkarte-farbe-pk50.noscale,ch.swisstopo.hangneigung-ueber_30,ch.swisstopo-karto.hangneigung,ch.bafu.wrz-jagdbanngebiete_select,ch.bazl.heliports-gebirgslandeplaetze,ch.swisstopo-karto.schneeschuhrouten,ch.swisstopo-karto.skitouren,ch.swisstopo.skitourenkarte-50.metadata,ch.bav.haltestellen-oev&layers_visibility=false,false,false,false,true,false,true,true,true,true,true,false,true&layers_timestamp=18641231,,,,,,,,,,,,&layers_opacity=1,1,1,1,0.85,0.5,0.2,0.6,1,0.8,0.8,1,1&catalogNodes=2352&lon=7.70568&lat=46.32534&elevation=6733&heading=359.700&pitch=-39.862)

**Comment by loicgasser** [comment]
LGTM

File: rc_prod [hunk 1]
```diff
@@ -1,6 +1,7 @@
 export KEEP_VERSION=true
 export DEPLOY_TARGET=prod
 export API_URL=//api3.geo.admin.ch
+export API_URL=//wmts{s}.geo.admin.ch
```
Comment (gjn) [inline rc_prod]: This is certainly wrong...

File: Makefile [hunk 2]
```diff
@@ -425,6 +436,7 @@ src/TemplateCacheModule.js: src/TemplateCacheModule.mako.js \
 apache/app.conf: apache/app.mako-dot-conf \
 	    ${MAKO_CMD} \
 	    .build-artefacts/last-api-url \
+	    .build-artefacts/last-mapproxy-url \
```
Comment (gjn) [inline Makefile]: why is this a dependency of the apache configuration?

File: Makefile [hunk 3]
```diff
@@ -311,7 +314,8 @@ prd/geoadmin.appcache: src/geoadmin.mako.appcache \
 	    --var "public_url=$(PUBLIC_URL)" $< > $@
 
 prd/cache/: .build-artefacts/last-version \
-			.build-artefacts/last-api-url
+			.build-artefacts/last-api-url \
+			.build-artefacts/last-mapproxy-url
```
Comment (gjn) [inline Makefile]: Same question here. I don't see why there's this dependency.

**Comment by gjn** [comment]
In principle, I agree. But I have some suspicion regarding implementation.

**Pull request merged**